### ISSUE TITLE
How to switch from "disabled" to "enabled" for textarea in Vue

### ISSUE THREAD

**Issue by WuDiXiaoShuaiCai** [ankurk91/vue-trumbowyg#26]
![image](https://user-images.githubusercontent.com/31613999/64933376-a015c580-d877-11e9-96fb-5d6ea0eba3fd.png)

**Comment by ankurk91** [comment]
The vue component does not watch for config options in realtime.

I found there is an option to disable the editor once initialised.
https://alex-d.github.io/Trumbowyg/documentation/#disabled

For now, you can use this workaround.
https://jsfiddle.net/afydcequ/1/

* add a ref to component
```html
<trumbowyg ref="editor" v-model="content"></trumbowyg
```
* call the method via ref
```js
this.$refs.editor.el.trumbowyg("disable")           
```

**Comment by WuDiXiaoShuaiCai** [comment]
thank you!

**Comment by ankurk91** [comment]
Found out that trumbowyg does not respect new config options once initialised. We need to destroy existing instance and re-init editor with new options.
I am not sure the side effects of this and how many people would get benefit.

**Comment by Laerolf** [comment]
@ankurk91, I have been trying out vue-trumbowyg and I think it is wonderfull.

The only issue that I encounter now is when I disable the component like you said above, even if I put a v-if on it, there is still an element left:

`<textarea data-v-ebb76330="" data-v-308c6a37="" class="" tabindex="-1" style=""></textarea>` 

How can I completly remove/destroy and initialized Trumbowyg?

**Comment by ankurk91** [comment]
Well, `v-if` should force the component to re-init.
I will try to reproduce it. 

You can destroy the editor instance like 
```
this.$refs.editor.el.trumbowyg("destroy")
```

**Comment by Laerolf** [comment]
Thank you for your effort, I also try to find out what the issue is by forking the project.
I tried the function like you provided, still doesn't work.

It seems like 'el' is always there.

**Comment by ankurk91** [comment]
Should be fixed in v3.5.0

**Comment by Laerolf** [comment]
Works really nicely thank you!
